learn/tools-ides/vscode-plugin/graphical-editor.md

```diff
@@ -8,7 +8,7 @@ redirect_from:
 
 # Graphical View
 
-A rich set of Visualization tools will immensely enhance your development experience especially in the integration space. 
+A rich set of visualization tools will immensely enhance your development experience especially in the integration space. 
```

Comment:
Shall we fix the same in the other 3 files as well?

Reply:
Fixed.

Reply:
Still cannot see this fix in the other files. Please recheck.

Reply:
Fixed.